dask/dataframe/tests/test_dataframe.py

```diff
@@ -1753,3 +1752,13 @@ def test_sorted_index_single_partition():
     ddf = dd.from_pandas(df, npartitions=1)
     eq(ddf.set_index('x', sorted=True),
         df.set_index('x'))
+
+
+def test_info(capsys):
+    df = pd.DataFrame({'x': [1, 2, 3, 4], 'y': [1, 0, 1, 0]})
+    ddf = dd.from_pandas(df, npartitions=1)
+    ddf.info()
+    out, err = capsys.readouterr()
+    assert out == "<class 'dask.dataframe.core.DataFrame'>" + '\n' + \
+                  "Data columns (total 2 columns):" + '\n' + \
+                  "x    int64" + '\n' + "y    int64"
```

Comment:
One nice feature of python is that strings wrapped in `()` are concatenated together on compilation. I find this to be slightly more readable. Again, not a big deal.

```python
assert out == ("<class 'dask.dataframe.core.DataFrame'>\n'
               "Data columns (total 2 columns):\n'
               "x    int64\n"
               "y    int64")   # Adding another `\n` at the end if you do my proposed fix above :)
```